_pages/capitalization.md

```diff
@@ -21,11 +21,19 @@ The following are capitalized:
 The following are not capitalized (unless at the start of a sentence):
 
 - deployment target
-- environment 
+- environment
 - server
 - built-in repository
 - runbooks (the concept not the feature)
 
+## Proper nouns vs common nouns
+
+A proper noun is the name of specific instance of a thing, Australia, Octopus Deploy. A common noun is general concept that refers to a class of things, country, deployment tool. The context words are used in can change whether a word is being used as a proper noun or a common noun. For instance,
```

Comment:
Opportunity to remove words.

Suggestion:
The context ~~words are used in~~ can change ...